hazelcast/src/test/java/com/hazelcast/internal/eviction/EvictionPolicyEvaluatorTest.java

```diff
@@ -125,20 +125,25 @@ public EvictionPolicyComparator getComparator() {
                 = new ArrayList<EvictionCandidate<Integer, CacheObjectRecord>>();
 
         long baseTime = System.currentTimeMillis();
-
+        long minCreationTime = -1;
         for (int i = 0; i < recordCount; i++) {
             long creationTime = baseTime + (i * 100);
+            minCreationTime = minCreationTime == -1 ? creationTime : Math.min(creationTime, minCreationTime);
```

Comment:
nit: could also initialize `minCreationTime = Long.MAX_VALUE` and just use `minCreationTime = Math.min(creationTime, minCreationTime);` here instead of the ternary operator.